app/services/isolate.rb

```diff
@@ -155,12 +155,13 @@ def exist? filename
 
   # cleans the box directory of any files by re-initializing
   def clean!
+    system "isolate -b#{@box_id} --cleanup #{"--cg" if has_cgroup?}", :out => '/dev/null'
     system "isolate -b#{@box_id} --init #{"--cg" if has_cgroup?}", :out => '/dev/null'
     init_boxdir()
   end
 
   def expand_path filename
-    File.expand_path(filename,"/tmp/box/#{@box_id}/box")
+    File.expand_path(filename,"/var/local/lib/isolate/#{@box_id}/box")
```

Comment:
Default box directory was changed in https://github.com/ioi/isolate/commit/8af30e70191972a105e457920af9e6cca99654cb.